### DISCUSSION TITLE
how to fix FPS camera clipping

### DISCUSSION THREAD

**Discussion by asumadelta** [Sixze/ALS-Refactored#92] [Q&A] [answered]
Hi I test ALS FPS camera and found out that only CPP version  camera clipping into the gun or hand if near

how can I fix this ?

![Untitled](https://user-images.githubusercontent.com/23005914/174469337-35b930f5-80ee-413f-be1f-bc25650d0177.png)

**Comment by Sixze** [comment, marked as answer]
Tweak this value in the project settings:

![image](https://user-images.githubusercontent.com/16257871/174608049-641ab6f9-6a26-4f17-b7aa-cb23a3902633.png)

**Reply by asumadelta** [reply]
seem like it not update when I adjust, I have to use CMD but yes this is solution thank you.
